admin/create_test_user.py

```
@@ -0,0 +1,46 @@
+
+    try:
+        # Check if user already exists
+        existing_user = db_manager.get_admin_user(username)
+        if existing_user:
+            print(f"User '{username}' already exists!")
+            return
+
+        # Create the user
+        user_id = auth_manager.create_admin_user(username=username, password=password, email=email, role=role)
+
+        print(f"✅ Successfully created admin user:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)